pages/start.tsx

```diff
@@ -70,7 +68,7 @@ const LoginStep1: React.FC<LoginStep1Props> = ({ user }) => {
     } else if (onboarded === false && user && providerToken) {
       setQueryParams({ step: "2" } satisfies QueryParams);
     }
-  }, [user, onboarded, providerToken]);
+  }, [user, onboarded, providerToken, router]);
```

Comment:
did this get added because of a linter warning?

Reply:
Yes, I can take it out though. I thought it was an error

Reply:
It's because it's used in the useEffect. Was just checking is all. Feel free to keep it in.